### DISCUSSION TITLE
Prepare Joomla 4 for TinyMCE 6.0

### DISCUSSION THREAD

**Discussion by dgrammatiko** [joomla/joomla-cms#35891] [Q&A] [answered]
As per tinyMCE's docs: https://www.tiny.cloud/docs/release-notes/6.0-upcoming-changes/ there are some changes that need to be done for moving to version 6.0.

Just leaving this here...

**Comment by brianteeman** [comment]
quick look and it doesn't look like too much to change for our implementation

**Comment by dgrammatiko** [comment]
@bembelimen this is probably something you want to follow for 4.1

**Comment by brianteeman** [comment, marked as answer]
I think I have addressed everything in #35889
